src/main/charts/bamboo/templates/statefulset.yaml

```diff
@@ -0,0 +1,160 @@
+            - name: ATL_TOMCAT_SCHEME
+              value: "https"
+            - name: ATL_TOMCAT_SECURE
+              value: "true"
+            {{ end }}
+            {{ if .Values.bamboo.service.contextPath }}
+            - name: ATL_TOMCAT_CONTEXTPATH
+              value: {{ .Values.bamboo.service.contextPath | quote }}
+            {{ end }}
+            {{ if .Values.ingress.host }}
+            - name: ATL_PROXY_NAME
+              value: {{ .Values.ingress.host | quote }}
+            - name: ATL_PROXY_PORT
+              value: {{ include "bamboo.ingressPort" . | quote }}
+            {{ end }}
+            {{- include "bamboo.databaseEnvVars" . | nindent 12 }}
+            {{/* TODO: This probably isnt needed with the current active-passive Bamboo architecture "clustering" does not work as we know it i.e. active-active*/}}
+            {{- include "bamboo.clusteringEnvVars" . | nindent 12 }}
+            {{ with .Values.bamboo.license.secretName }}
+            {{/* TODO: Bamboo does not currently support this. Keeping for now as support may come later.*/}}
+            - name: BAMBOO_SETUP_LICENSE
+              valueFrom:
+                secretKeyRef:
+                  name: {{ . }}
+                  key: {{ $.Values.bamboo.license.secretKey }}
+            {{ end }}
+            - name: SET_PERMISSIONS
+              value: {{ .Values.bamboo.setPermissions | quote }}
+            - name: BAMBOO_SHARED_HOME
+              value: {{ .Values.volumes.sharedHome.mountPath | quote }}
+            - name: JVM_SUPPORT_RECOMMENDED_ARGS
+              valueFrom:
+                configMapKeyRef:
+                  key: additional_jvm_args
+                  name: {{ include "bamboo.fullname" . }}-jvm-config
+            - name: JVM_MINIMUM_MEMORY
+              valueFrom:
+                configMapKeyRef:
+                  key: min_heap
+                  name: {{ include "bamboo.fullname" . }}-jvm-config
+            - name: JVM_MAXIMUM_MEMORY
+              valueFrom:
+                configMapKeyRef:
+                  key: max_heap
+                  name: {{ include "bamboo.fullname" . }}-jvm-config            
+            {{- include "bamboo.additionalEnvironmentVariables" . | nindent 12 }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.bamboo.ports.http }}
+              protocol: TCP
+            - name: jms
+              containerPort: {{ .Values.bamboo.ports.jms }}
+              protocol: TCP
+{{/*          TODO: Need to re-think the readiness probe if we decide for first release that running more than 1 server is */}}
+{{/*          acceptable with the current bamboo architecture.*/}}
+{{/*           */}}
+{{/*          When a second bamboo server (passive) is added to the cluster, the readiness probe fails. This is because */}}
+{{/*          the active pod never fully comes up on account of the cluster lock being held by the active node.*/}}
```

Comment:
Can probably remove this at some stage as active-passive (1 pod) is likely what we will ship with